pymysqlreplication/binlogstream.py

```diff
@@ -142,7 +143,8 @@ def __init__(self, connection_settings, server_id,
                  slave_heartbeat=None,
                  is_mariadb=False,
                  annotate_rows_event=False,
-                 ignore_decode_errors=False):
+                 ignore_decode_errors=False,
+                 parma_logging=True, ):
```

Comment:
I suggest to document it. Maybe we can find a name more explicit also but I have nothing in mind.